modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -256,20 +256,20 @@ If the IdP declares a redirect URI, it might override the target URL (`redirect_
 
 == Configure logout behaviour
 
-If your Bonita platform is configured to manage authentication over OIDC, when users log out of Bonita Portal, they do not log out of the OIDC Identity Provider (IdP).
+If your Bonita platform is configured to manage authentication over OIDC, when users log out of Bonita Runtime, they do not log out of the OIDC Identity Provider (IdP).
 Therefore they are not logged out of all applications that are using the OIDC provider.
 To avoid this, you have two options :
 
 [discrete]
-==== Hide the logout button of the portal
+==== Hide the logout button of the Bonita Layout
 
 This is the most commonly used solution. Users are logged in as long as they don't close their web browser (unless their session times out).
```

Comment:
Users are logged in as long as they don't close their web browser (unless their session times out). => Users are logged in as long as they don't close their web browser and the session does not time out.

Reply:
I added this with a previous comment